dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/TrackerTrackedEntityCriteriaMapper.java

```diff
@@ -216,7 +220,7 @@ private Set<DimensionalItemObject> getDuplicatedAttributes( List<QueryItem> attr
             .collect( Collectors.toSet() );
     }
 
-    private Set<OrganisationUnit> validateOrgUnits( Set<String> orgUnitIds, User user )
+    private Set<OrganisationUnit> validateOrgUnits( Set<String> orgUnitIds, User user, Program program )
```

Comment:
nit: I would put the user first to show we validate that this user has access to the rest of the parameters (entities).